test/Test/Pos/Types/BlockSpec.hs

```diff
@@ -50,10 +50,7 @@ spec = describe "Block properties" $ do
     verifyEmptyHsDesc = "Successfully validates an empty header chain"
     emptyHeaderChain l =
         it verifyEmptyHsDesc $
-            and
-                [isVerSuccess $
-                    T.verifyHeaders b l
-                        | b <- [False, True]] == True
+            and [isVerSuccess $ T.verifyHeaders b l | b <- [False, True]]
```

Comment:
I'd further replace it with `all isVerSuccess [T.verifyHeaders b l | b <- [False, True]]`